MPCAutofill/cardpicker/views.py

```diff
@@ -196,6 +199,68 @@ def contributions(request: HttpRequest) -> HttpResponse:
     return render(request, "cardpicker/contributions.html", {"sources": sources, "total_count": total_count_f})
 
 
+def patrons(request: HttpRequest) -> HttpResponse:
+    # Disable page without Patreon
+    if not PATREON_ENABLED:

```

Comment:
isn't this setting redundant? you can just check if `PATREON_KEY` and `PATREON_URL` both have length > 0

Reply:
It certainly is, my thinking at the time was to have an unambiguous toggle for clarity's sake, but rethinking it I'll just remove this setting, check for an api on the backend, check for url on the front end. Good call

Reply:
Done!